pom.xml

```diff
@@ -51,6 +51,7 @@
     <sqlline.version>1.1.9-drill-r7</sqlline.version>
     <jackson.version>2.9.5</jackson.version>
     <jackson.databind.version>2.9.5</jackson.databind.version>
+    <zookeeper.version>3.4.11</zookeeper.version>
```

Comment:
Why do not use last stable version 3.4.12?
https://issues.apache.org/jira/browse/DRILL-6534?focusedCommentId=16522622&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16522622